packages/dx-demo-shell/generate-demos.js

```diff
@@ -121,18 +136,31 @@ const overrideFileIfChanged = (filename, data) => {
 };
 const createFromTemplate = (sourceFilename, outputFilename, data) => {
   const source = fs.readFileSync(sourceFilename, 'utf-8');
-  mustache.tags = ['<%', '%>'];
   const output = mustache.render(source, data);
   overrideFileIfChanged(outputFilename, output);
   cancelFileRemoving(outputFilename);
 };
+const findSplitter = (content, index) => {
+  const k = content.indexOf(' from ', index + 1);
+  if (k >= 0) {
+    return findSplitter(content, k);
+  }
+  return index >= 0 ? content.indexOf('\n', index + 1) : -1;
+};
+const splitPartialContent = (content) => {
+  const index = findSplitter(content, -1);
```

Comment:
Maybe it'd be more clear if we used some splitter tag in partial templates

Reply:
Yes, it would. But we will have to manually put slitter tag in all files.
We have to choose.

Let everybody vote here.

Reply:
+1 to use slitter tag

Reply:
Now each block is expected to be wrapped with 
```jsx
// BLOCK:<block-name>
...
// BLOCK:<block-name>
```
and then referenced as `<%&block-name%>`.